src/Connection/Http1Connection.php

```diff
@@ -92,7 +92,7 @@ public function __construct(EncryptableSocket $socket, int $timeoutGracePeriod =
         $this->remoteAddress = $socket->getRemoteAddress();
         $this->tlsInfo = $socket->getTlsInfo();
         $this->timeoutGracePeriod = $timeoutGracePeriod;
-        $this->lastUsedAt = getCurrentTime();
+        $this->lastUsedAt = Loop::now();
```

Comment:
Why `Loop::now()` instead of `getCurrentTime()`?

Reply:
Only because `Loop::now()` caches it during a tick.

Reply:
I think this is rather a disadvantage, because timeouts will be shorter on a busy loop then.

Reply:
I switched it to `getCurrentTime()`.